editor/scene_tree_dock.cpp

```diff
@@ -1843,7 +1846,6 @@ bool SceneTreeDock::_check_node_path_recursive(Node *p_root_node, Variant &r_var
 			}
 			break;
 		};
```

Comment:
Just a consistency nitpick.
```suggestion
		} break;
```